docs/naps/0-nap-process.md

```diff
@@ -0,0 +1,297 @@
+
+Wherever possible, it is considered preferable to reduce the scope of a
+proposal to avoid the need to rely on the "Provisional" status (e.g. by
+deferring some features to later NAPs), as this status can lead to version
+compatibility challenges in the wider ecosystem.
+
+A NAP can also be assigned status `Deferred`. The NAP author or a core
+developer can assign the NAP this status when no progress is being made on
+the NAP.
+
+A NAP can also be `Rejected`. Perhaps after all is said and done it
+was not a good idea. It is still important to have a record of this
+fact. The `Withdrawn` status is similar---it means that the NAP author
+themselves has decided that the NAP is actually a bad idea, or has
+accepted that a competing proposal is a better alternative.
+
+When a NAP is `Accepted`, `Rejected`, or `Withdrawn`, the NAP should be
+updated accordingly. In addition to updating the status field, at the very
+least the `Resolution` header should be added with a link to the relevant
+post on the discussion forum.
+
+NAPs can also be `Superseded` by a different NAP, rendering the
+original obsolete. The `Replaced-By` and `Replaces` headers
+should be added to the original and new NAPs respectively.
+
+Process NAPs may also have a status of `Active` if they are never
+meant to be completed, e.g. NAP 0 (this NAP).
+
+### How a NAP becomes Accepted
+
+A NAP is `Accepted` by consensus of all interested contributors. We need a
+concrete way to tell whether consensus has been reached. When you think a
+NAP is ready to accept, start a topic on the Zulip naps channel with a
+subject like:
+
+> Proposal to accept NAP #\<number>: \<title>
+
+In the body of the topic, you should:
+
+- link to the latest version of the NAP,
+- briefly describe any major points of contention and how they were
+  resolved,
+- include a sentence like: "If there are no substantive objections
+  within 7 days from this post, then the NAP will be accepted; see
+  NAP 0 for more details."
+
+For an equivalent example in the NumPy library, see: <https://mail.python.org/pipermail/numpy-discussion/2018-June/078345.html>
+
+After you write the post, you should make sure to link to the specific
+thread from the `Discussion` section of the NAP, so that people can
+find it later.
+
+Generally the NAP author will be the one to make this post, but
+anyone can do it – the important thing is to make sure that everyone
+knows when a NAP is on the verge of acceptance, and give them a final
+chance to respond. If there's some special reason to extend this final
+comment period beyond 7 days, then that's fine, just say so in the
+post. You shouldn't do less than 7 days, because sometimes people are
+traveling or similar and need some time to respond.
+
+In general, the goal is to make sure that the community has consensus,
+not provide a rigid policy for people to try to game. When in doubt,
+err on the side of asking for more feedback and looking for
```

Comment:
👏

Reply:
Ah ok - I might put this higher, set the tone a little bit right from the start